.github/workflows/back-tests.yml

```
@@ -3,18 +3,26 @@ name: Backend Tests
+    branches: [main, dev]
   workflow_dispatch:
 
 jobs:
-  test:
+  changes-check:
     runs-on: ubuntu-latest
+    outputs:
+      any_changed: ${{ steps.filter.outputs.any_changed }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dorny/paths-filter@v2
```

actionlint (1.7.4)

the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)